src/api/entities/SecurityToken/index.ts

```diff
@@ -319,4 +322,28 @@ export class SecurityToken extends Entity<UniqueIdentifiers> {
 
     return boolToBoolean(result);
   }
+
+  /**
+   * Modify the primary issuance agent of the Security Token
```

Comment:
Missing param descriptions (for the expiry you should be explicit that it's for the authorization and not for the primary issuance agent itself)